generators/server/files.js

```diff
@@ -82,6 +85,11 @@ const serverFiles = {
         {
             path: SERVER_TEST_RES_DIR,
             templates: ['application.properties']
+        },
+        {
+            condition: generator => generator.authenticationType === 'oauth2',
+            path: SERVER_TEST_RES_DIR,
+            templates: [{ file: 'jhipster-test-realm.json', method: 'copy', noEjs: true }]
```

Comment:
How is this different from the regular one? We only have one for Spring Security.

Reply:
Removed via the Mock Oidc Test Resource